native/src/main/java/io/ballerina/stdlib/graphql/runtime/engine/Engine.java

```diff
@@ -236,4 +242,22 @@ public static BString getInterceptorName(BObject interceptor) {
         ServiceType serviceType = (ServiceType) interceptor.getType();
         return StringUtils.fromString(serviceType.getName());
     }
+
+    public static Object getResourceAnnotation(BObject service, BString operationType, BArray path,
+                                               BString methodName) {
+        ServiceType serviceType = (ServiceType) service.getType();
+        BString identifier = StringUtils.fromString(getModule().toString() + COLON + RESOURCE_CONFIG);
```

Comment:
Just checking will this return the correct module prefix if we use import with alias (as) ?

Reply:
Used the same util function that was used in other places.